src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -25,11 +37,36 @@ export enum TableType {
   FAVOURITE = 'FAVOURITE',
 }
 
+const _filterCb = (token: Token, query?: string) => {
+  if (!query) return false
+
+  const cleanQuery = query.toLowerCase()
+  const address = token.address.toLowerCase()
+  const symbol = token.symbol?.toLowerCase()
+  const name = token.name?.toLowerCase()
+
+  return address.match(cleanQuery) || symbol?.match(cleanQuery) || name?.match(cleanQuery)
+}
+
 export default function TokenTable({
-  tokensData,
+  tokensData: rawTokensData,
   maxItems = MAX_ITEMS,
   tableType = TableType.OVERVIEW,
 }: TokenTableParams) {
+  // search - takes precedence re:filtering
+  const [query, setQuery] = useState<string>()
```

Comment:
I think we should set an empty string as a default value here, otherwise you get this warning 
![Screenshot from 2022-06-08 14-00-11](https://user-images.githubusercontent.com/34926005/172611420-b562030f-79d2-4065-8802-33dbdff8a6c2.png)